_episodes/02-shell_to_cwl.md

```diff
@@ -27,792 +27,215 @@ keypoints:
 
-baseCommand: echo
-inputs:
-  message_text:
-    type: string
-    default: 'hello World!'
-    inputBinding:
-      position: 1
+The output displayed above shows us that the program has run succesfully and what its output is, `Hello world!`. 
 
-outputs:
-  message_out:
-    type: stdout
+Let's take a look at the `echo.cwl` script in more detail.
 
-~~~
-{: .language-yaml}
-Here we have added the `message_out` item, which has been given type `stdout` (as we want to capture the output at the command line, rather than any particular files).
+As explained above, the first 2 lines are always the same. You define the CWL version and which class you are using. In this example we have a `CommandLineTool`, in particular the `echo` command.
+The next line, `baseCommand`, contains the command we want to run (`echo`). 
 
-Now we run the script:
-~~~
-$ cwltool echo.cwl moon.yml
 ~~~
-{: .language-bash}
-~~~
-INFO .../cwltool
-INFO Resolved 'echo.cwl' to 'file:///.../echo.cwl'
-INFO [job echo.cwl] /private/tmp/docker_tmp7k0bqeg5$ echo \
-    'hello Moon!' > /private/tmp/docker_tmp7k0bqeg5/9611f21693018fe4ce4bf1f3884e47dae2ce3aab
-INFO [job echo.cwl] completed success
-{
-    "message_out": {
-        "location": "file:///.../9611f21693018fe4ce4bf1f3884e47dae2ce3aab",
-        "basename": "9611f21693018fe4ce4bf1f3884e47dae2ce3aab",
-        "class": "File",
-        "checksum": "sha1$d4413a97a36059e8855168ac7939a4cb5d4da9c9",
-        "size": 12,
-        "path": "/.../9611f21693018fe4ce4bf1f3884e47dae2ce3aab"
-    }
-}
-INFO Final process status is success
-~~~
-{: .output}
-Note that the output has been saved as a file called `9611f21693018fe4ce4bf1f3884e47dae2ce3aab` (not `message_out`, which is only the output variable name) in this instance. When you run this script the file name will be different. You can open this text file with a text editor to confirm that it contains the expected message.
-
-It is not very user-friendly to have our script return a randomly named file each time, so we will make use of the `stdout` field to specify the name of the text file that we want standard output to be captured to:
-~~~
-cwlVersion: v1.2
-class: CommandLineTool
-
-baseCommand: echo
 inputs:
   message_text:
     type: string
-    default: 'hello World!'
-    inputBinding:
-      position: 1
-
-stdout: output.txt
-
-outputs:
-  message_out:
-    type: stdout
+	inputBinding:
+	  position: 1
 ~~~
 {: .language-yaml}
 
-Running the script will produce this output:
-~~~
-$ cwltool echo.cwl moon.yml
-~~~
-{: .language-bash}
+This block of code contains the `inputs` section of our workflow. In this section we provide all the inputs that are needed for the workflow. 
+In this example a string is needed that will be printed on the command line. We first define an id for our input: `message_text`. 
+Then we need to define the type of the input. In our case it will be a `string`. The `InputBinding` describes how the input should appear on the command line.
```

Comment:
Did "we" write this? Since we didn't, maybe switch around the wording

Reply:
I've changed this in the latest commit (https://github.com/carpentries-incubator/cwl-novice-tutorial/pull/53/commits/86a46d2efaf811c46c7ef3b4e80fbed1783156cf)

Reply:
I must admit my writing style here was more informal/conversational - this does follow the general style that the software carpentry courses use though, e.g.: https://swcarpentry.github.io/shell-novice/02-filedir/index.html

I'm agnostic on which is a better style to use though. The software carpentry material is intended to be read out by an instructor to a class, so perhaps in that environment, conversational language is more fitting. I guess it depends on how you see the material being used, and who your target audience is?

Reply:
Repeating what I discussed with @ALuesink elsewhere, this particular section is about reading an already written CWL tool description, not actively writing a new one. So no action verb phrases "we first define"; nothing is being written from scratch.

I am fully supportive of a conversational tone over a formal tone! :-)